metno/__init__.py

```diff
@@ -101,12 +101,12 @@ def get_forecast(self, time_zone):
         if self.data is None:
             return []
 
-        now = datetime.datetime.now(time_zone).replace(hour=12, minute=0,
+        now = datetime.datetime.now(time_zone).replace(hour=0, minute=0,
```

Comment:
Then the forecast will be for midnight, and not for 12:00.

Reply:
Yeah, that's a problem. To be honest I haven't completely understood yet how the forecast for a particular day is given. Is the idea maybe that if `max_hour` was set to 6, and hour (here) to 12, the available forecast closest to (but no more than 6 hours away from) hour 12 would be returned?